src/components/VisuallyHidden/VisuallyHidden.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+
+type VisuallyHiddenProps = {
+  text: string;
+};
+
+export const VisuallyHidden = ({ text }: VisuallyHiddenProps) => {
```

Comment:
@chriddell what are your thoughts on this?

I like the accessibility but I'm not sure if it's worth having a React component for this. I'm not completely opposed though.

If we decide it is worth having this component then the space should be added in the `text` prop and not in the component since it might be used elsewhere in a sentence.

Reply:
I think it could be useful to have a component for this, actually...

- easier to change CSS selector (e.g. `u-visually-hidden` can become `u-hide-visually` in a one line change)
- approach to hiding content becomes explicitly consistent

One thing I would suggest is using a `children` prop rather than `text` prop, to allow the component to render either other React components or HTML elements, e.g.

```
<VisuallyHidden wrapperAs="div">
  <h2>Title of the long text</h2>
  <p>This is a longer piece of text which is hidden</p>
</VisuallyHidden>
```

Reply:
I have updated `VisuallyHidden` component :)